pkgs/by-name/al/alacritty/package.nix

```diff
@@ -43,16 +44,16 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "alacritty";
-  version = "0.14.0";
+  version = "0.15.0";
 
   src = fetchFromGitHub {
     owner = "alacritty";
     repo = pname;
```

Comment:
```suggestion
    repo = "alacritty";
```

Reply:
If you have nitpick changes like that, could you just push them directly to the branch in question? This saves another review round trip on both ends.